front/ui/ui-charts/src/spaceTimeChart/components/PathLayer.tsx

```diff
@@ -451,8 +454,63 @@ export const PathLayer = ({
     [border, getPathLines, level]
   );
 
+  const drawSinglePoint = useCallback<DrawingFunction>(
+    (
+      ctx,
+      {
+        getTimePixel,
+        getSpacePixel,
+        hidePathsLabels,
+        theme: {
+          pathsStyles: { fontSize, fontFamily },
+        },
+      }
+    ) => {
+      if (path.points.length !== 1) return;
```

Comment:
nit: 
don't think we could have this situation but maybe is it safer with:
```suggestion
      if (path.points.length > 1) return;
```

Reply:
the goal is to execute the code only when there is exactly one point.

Your proposal `(> 1)` would also allow the case where `path.points.length === 0`, but in that case the code would still try to access `path.points[0]` and could lead to unexpected behavior or errors.

That’s why using `!== 1 `is safer here: it makes sure we only enter the function when **there’s exactly one point** and return early for all other cases.